src/test/java/org/owasp/esapi/logging/java/JavaLogLevelHandlersTest.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * OWASP Enterprise Security API (ESAPI)
+ * 
+ * This file is part of the Open Web Application Security Project (OWASP)
+ * Enterprise Security API (ESAPI) project. For details, please see
+ * <a href="http://www.owasp.org/index.php/ESAPI">http://www.owasp.org/index.php/ESAPI</a>.
+ *
+ * Copyright (c) 2007 - The OWASP Foundation
+ * 
+ * The ESAPI is published by OWASP under the BSD license. You should read and accept the
+ * LICENSE before you use, modify, and/or redistribute this software.
+ * 
+ * @created 2018
+ */
+package org.owasp.esapi.logging.java;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+import org.mockito.Mockito;
+
+public class JavaLogLevelHandlersTest {
```

Comment:
All the handler tests are duplicated test blocks.  Look into building parameterized tests to reduce duplication and mainteance.

Reply:
Low priority.  Can be assessed outside of initial integration